dist/images/ovnkube.sh

```diff
@@ -981,6 +984,11 @@ local-nb-ovsdb() {
   ovn-nbctl set NB_Global . name=${K8S_NODE}
   ovn-nbctl set NB_Global . options:name=${K8S_NODE}
 
+ [[ "true" == "${ENABLE_IPSEC}" ]] && {
+    ovn-nbctl set nb_global . ipsec=true
+    echo "=============== nb-ovsdb ========== reconfigured for ipsec"
```

Comment:
Why reconfigured ?

Reply:
this was needed for IC cluster, otherwise IPsec is not configured in nbdb.